pkg/kubectl/apps/kind_visitor.go

```diff
@@ -39,35 +39,31 @@ type GroupKindElement schema.GroupKind
 
 // Accept calls the Visit method on visitor that corresponds to elem's Kind
 func (elem GroupKindElement) Accept(visitor KindVisitor) error {
-	if elem.GroupMatch("apps", "extensions") && elem.Kind == "DaemonSet" {
+	switch {
+	case elem.GroupMatch("apps", "extensions") && elem.Kind == "DaemonSet":
 		visitor.VisitDaemonSet(elem)
 		return nil
-	}
-	if elem.GroupMatch("apps", "extensions") && elem.Kind == "Deployment" {
+	case elem.GroupMatch("apps", "extensions") && elem.Kind == "Deployment":
```

Comment:
the return nil can go to the end.

Reply:
Got it and Thanks

Reply:
@sttts Because switch has default return branch, the return nil can't go to the end.